Release note updates (#14691)
istio/istio.io: content/en/news/releases/1.21.x/announcing-1.21/_index.md

linktitle: 1.21.0
subtitle: Major Release
description: Istio 1.21 Release Announcement.

> publishdate: 2024-03-08

Reviewer: ```suggestion
publishdate: 2024-03-11
```

workload. Coming in at 25% smaller, the sidecar image can be pulled faster improving pod
startup times. Additionally, the reduced binary size typically results in a 5MB RAM
reduction - across many pods, this quickly adds up to cost savings.

### Support for all CNIs in ambient mode

> Our new [ambient mode](/blog/2022/introducing-ambient-mesh/) now works across all Kubernetes platforms and CNI implementations, Ambient mode has been tested with GKE, AKS, and EKS and all the CNI implementations they offer, as well as with 3rd-party CNIs like Calico and Cilium, as well as platforms like OpenShift, with solid results. The engineering challenges behind this fix were described in [a recent blog post](/blog/2024/inpod-traffic-redirection-ambient/).

Reviewer: ```suggestion
Our new [ambient mode](/blog/2022/introducing-ambient-mesh/) now works across all Kubernetes platforms and CNI implementations. Ambient mode has been tested with GKE, AKS, and EKS and all the CNI implementations they offer, 3rd-party CNIs like Calico and Cilium, and platforms like OpenShift, all with solid results. The engineering challenges behind this fix were described in [a recent blog post](/blog/2024/inpod-traffic-redirection-ambient/).
```

> Ambient mode is scheduled to move to Beta in the upcoming Istio 1.22.

Reviewer: ```suggestion
Ambient mode's Layer 4 is scheduled to move to Beta in the upcoming Istio 1.22.
```

Layer 7 will still be alpha as of now.

Author: We're going to market "ambient" as beta as a whole, so while technically correct, we'll leave this out for now.

Reviewer: Just want to confirm so that we're sending a uniform message, as John said something different https://github.com/istio/istio/pull/49644#issuecomment-1979498018. cc: @howardjohn

Author: We discussed this at Steering today and landed with being OK to say "targeted".

Reviewer: Just wanted to verify

## Upgrading to 1.21

> We would like to hear from you regarding your experience upgrading to Istio 1.20. You can provide feedback

Reviewer: ```suggestion
We would like to hear from you regarding your experience upgrading to Istio 1.21. You can provide feedback
```

Author: I knew I'd miss one
